tests/cluster/tests/21-many-slot-migration.tcl

```diff
@@ -1,10 +1,5 @@
 # Tests for many simultaneous migrations.
 
-# TODO: Test is currently disabled until it is stabilized (fixing the test
-# itself or real issues in the server).
-
-if {false} {
```

Comment:
Being able to re-enable these tests is a really good indicator that this improvement is good stuff.